### ISSUE TITLE
How to divide the training/test set on the M3FD dataset?

### ISSUE THREAD

**Issue by Zhangjunyin** [LidongWang-26/DiffV2IR#7]
As your mentioned in paper, images with same scene will pose a risk of data leakage, can you provide some details for a manual split of the m3fd dataset?

**Comment by LidongWang-26** [comment]
I put all the images from the same video sequence into training set or testing set, I will update my split of M3FD in the dataset link.

**Comment by LidongWang-26** [comment]
> At the same time, I also have another question. In the second phase of pretraining, 70,000 pairs of infrared and visible light data are used. However, this includes all the data samples from M3FD. Does this still constitute a data leakage issue?"

Data samples from M3FD are not used in the training data of second phase of pretraining when conducting experiments on M3FD. 
Data samples from FLIR-aligned are also not used in the training data of second phase of pretraining when conducting experiments on FLIR-aligned.

**Comment by Zhangjunyin** [comment]
Thank you very much for your reply. Could you please share the train/validation/test splits for the FLIR-aligned dataset, and also make available the scripts or files used to compute the evaluation metrics (e.g., SSIM, which is sensitive to window size)? Furthermore, if you could release the generated results on both M3FD and FLIR-aligned for the methods listed in Table 2, it would greatly facilitate future research and fair comparisons.

**Comment by LiYangBa** [comment]
> > 感谢您的回复。M3FD 数据集拆分的详细信息是否已上传到数据集链接？您能提供具体的链接吗？谢谢！
> 
> 已上传至 Quark。

Could you please share the download link? Thank you very much!

**Comment by EliPassov** [comment]
Hi, thank you very much for creating the dataset. 
Would also appreciate if you can upload the train/val/test split to HuggingFace
Also were the rest of the dataset (or at least the 70K paired data) split into train/val/test please?

In addition, it seems KAIST images were uploaded as .jpg files, but they are actually .png files (can open separate issue on that if you like)
Thank you.

**Comment by Kartikeya0809** [comment]
> > Thanks for your reply. Have the details of the M3FD dataset split been uploaded to the dataset link? Could you please provide the exact link? Thanks!
> 
> Uploaded in Quark.

Could you also upload the train and test splits for the FLIR-aligned dataset on HuggingFace?
